### DISCUSSION TITLE
Primefaces Designer Stuck on Setting Up

### DISCUSSION THREAD

**Discussion by redz09123** [primefaces/community#2323] [Figma UI Kit]
I am planning to customize a theme for the web app I am currently working on, but whenever I open the designer and pick the base theme, it gets stuck on 'setting up.' Is there currently a problem with the Prime Faces designer?

below is the image:
![image](https://github.com/user-attachments/assets/fae82e15-9083-4409-a720-be7d30e52d81)

**Comment by melloware** [comment]
Duplicate: https://github.com/orgs/primefaces/discussions/2315
